lib/fortnox/api/mappers/base/from_json.rb

```diff
@@ -45,7 +45,7 @@ def convert_collection( key, collection )
             # add it before a new release.
 
             raise MissingModelOrMapperException, "for #{ key } (#{ mapper_name }, #{ Fortnox::API::Mapper::DefaultTemplates.canonical_name_sym }) with #{ collection }" if ENV['RUBY_ENV']
-            Fortnox::API.logger.warn( "Missing Model or Mapper implementation for #{ key } with attributes: #{ collection }" )
+            Fortnox::API.config.logger.warn( "Missing Model or Mapper implementation for #{ key } with attributes: #{ collection }" )
```

Comment:
You could have aliased this on the root module instead (made logger there a class method that looks up the logger in the config).